js_modules/dagster-ui/packages/app-oss/src/pages/_document.tsx

```diff
@@ -50,18 +50,18 @@ export default function Document() {
         />
         <link
           rel="manifest"
-          href={`${process.env.NEXT_PUBLIC_URL ?? ''}/manifest.json`}
+          href={`${process.env.NEXT_PUBLIC_URL ?? PREFIX_PLACEHOLDER}/manifest.json`}
```

Comment:
I think this should be `${process.env.NEXT_PUBLIC_URL}/${PREFIX_PLACEHOLDER}/manifest.json`, can you confirm?

Reply:
I made the requested change!